src/main/java/com/spotify/confidence/Confidence.java

```diff
@@ -98,6 +104,86 @@ public void send(String eventName, ConfidenceValue.Struct message) {
     }
   }
 
+  public <T> T getValue(String key, T defaultValue) {
+    return getEvaluation(key, defaultValue).getValue();
+  }
+
+  public <T> FlagEvaluation<T> getEvaluation(String key, T defaultValue) {
+    try {
+      final FlagPath flagPath = SdkUtils.getPath(key);
+      final String requestFlagName = "flags/" + flagPath.getFlag();
+      final ResolveFlagsResponse response = resolveFlags(requestFlagName).get();
+      if (response.getResolvedFlagsList().isEmpty()) {
+        final String errorMessage =
+            String.format("No active flag '%s' was found", flagPath.getFlag());
+        log.warn(errorMessage);
+        return new FlagEvaluation<>(
+            defaultValue, "", "ERROR", ErrorType.FLAG_NOT_FOUND, errorMessage);
+      }
+
+      final String responseFlagName = response.getResolvedFlags(0).getFlag();
```

Comment:
Nit, get the `ResolvedFlag` variable already here and use it for the flagName.
```suggestion
      final ResolvedFlag resolvedFlag = response.getResolvedFlags(0);
```